docs/advanced-config.md

```diff
@@ -75,3 +75,30 @@ The following config options can be set by passing them as arguments to `startTr
    - `node_version`: `process.versions.node`, e.g. `16.10.0`
 
 \*: Overwritten default value
+
+### Start all
+
+To control all signals with one call `start()` API can be used:
```

Comment:
Is `signal` used elsewhere? Was confused by this word when I first saw it in this context

Reply:
It's widely used in OpenTelemetry: [Glossary](https://github.com/open-telemetry/opentelemetry-specification/blob/70fecd2dcba505b3ac3a7cb1851f947047743d24/specification/glossary.md#signals), [Overview](https://github.com/open-telemetry/opentelemetry-specification/blob/0c793b450098e8d7a739e575897707f7db08d8ce/specification/overview.md).

Do you know another term that's ever used as a noun to combine "tracing", "metrics", etc?

Reply:
It's fine, perhaps make `signals` link to the glossary here